### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by christianonoh**
# Implementing React events for Calculator component
This pull request includes the implementation of React events in the Calculator component using math logic from `calculate.js` and `operate.js`.

## Changes Made
- Created a new directory called logic inside the project
- Copied the files `calculate.js` and `operate.js` into the logic/ directory
- Installed `big.js` using NPM
- Imported `calculate.js` in the `Calculator.js` component
- Implemented click event handlers for all calculator buttons
- Tested the app in the browser to ensure that the Calculator can perform math operations

### CODE REVIEW

File: src/components/Buttons.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (CollinsTatang) [inline src/components/Buttons.js]: Great job so far @christianonoh 
- Please do not disable your linters fix it.
Please check the other files and ensure you don't disable any linters checks. Linters actions help us to write clean and international standard code 

Thank You

**Review by CollinsTatang — COMMENTED** [review]
Hi @christianonoh , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has Linters checks disabled.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,42 +1,67 @@
...
-      <Screen />
+      <Screen result={displayValue()} />
       <div className="buttons">
         {buttons.map((button) => (
           <Buttons
             key={button.value}
             value={button.value}
             className={button.className}
             name={button.name}
+            handleClick={handleClick}
```
Comment (abdulhamiid) [inline src/components/Calculator.js:64]: Currently, the calculator is working fine 💯 but I'm getting this error in the console.

image

Please kindly ensure that the app runs smoothly as it should without any errors. 🛠👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @christianonoh 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Linters checks are passing
- ✔️ Use Git Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @christianonoh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] `Calculator` app handles state with `useState` ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**